protocols/L1618/Swift16s.ot2.py

```diff
@@ -0,0 +1,303 @@
+
+    wash_count = 1
+
+    def wash(wvol, rvol):
+        nonlocal wash_count
+        m300.set_flow_rate(aspirate=50, dispense=50)
+
+        # add wash
+        pick_up(m300)
+        for samples in magcol:
+            m300.transfer(wvol, reagent_res.wells(wash_count), samples.top(-1),
+                          air_gap=20, new_tip='never')
+            m300.blow_out(samples.top(-1))
+        m300.delay(seconds=30)
+
+        m300.set_flow_rate(dispense=300)
+        for samples in magcol:
+            if not m300.tip_attached:
+                pick_up(m300)
+            m300.transfer(rvol, samples, etoh_waste.top(-1), air_gap=20,
+                          new_tip='never')
+            m300.blow_out(etoh_waste.top(-1))
+            m300.drop_tip()
+        m300.set_flow_rate(aspirate=150)
+        wash_count += 1
+
+    """ Multiplex PCR """
+    pick_up(s50)
+    s50.mix(10, 45, mm1)
+    s50.blow_out(mm1.top())
+    for sample in temp1:
+        if not s50.tip_attached:
+            pick_up(s50)
+        s50.transfer(20, mm1, sample, new_tip='never')
+        s50.blow_out(sample.top())
+        s50.drop_tip()
+
+    if number_of_samples == '96':
+        pick_up(s50)
+        s50.mix(10, 45, mm2)
+        s50.blow_out(mm2.top())
+        for sample in temp2:
+            if not s50.tip_attached:
+                pick_up(s50)
+            s50.transfer(20, mm2, sample, new_tip='never')
+            s50.blow_out(sample.top())
+            s50.drop_tip()
+
+    robot.pause('Please remove plate from the temperature module and place in \
+    thermocycler for PCR run. When nearing completion of PCR run, place the \
+    Index Reaction Mix in tube slots A3 (and A4 if doing 96 samples) in deck \
+    slot 3. After completion of PCR run, place sample on the magnetic module \
+    in deck slot 1 and click RESUME in OT App.')
+
+    """ Size Selection and Clean-Up Step 1 """
+    pick_up(m300)
+    m300.mix(5, 100, magbeads)
+    m300.blow_out(magbeads.top(-1))
+    magdeck.disengage()
+
+    for sample in magcol:
+        if not m300.tip_attached:
+            pick_up(m300)
+        m300.transfer(36, magbeads, sample, new_tip='never')
+        m300.blow_out(sample.top())
+        m300.drop_tip()
+
+    m300.delay(minutes=5)
+    robot._driver.run_flag.wait()
+    magdeck.engage()
+    m300.delay(minutes=5)
+
+    for sample in magcol:
+        pick_up(m300)
+        m300.transfer(60, sample, etoh_waste, new_tip='never')
+        m300.drop_tip()
+
+    wash(200, 200)
+
+    wash(200, 200)
+
+    magdeck.disengage()
+    robot.pause('Pulse-Spin samples in microfuge and place back on Magnetic \
+    Module. Before beginning Indexing Step, be sure Index D50X and D7XX are \
+    placed in the positions specified in the protocol documentation. Place \
+    the plate back on the Magnetic Module when ready and click RESUME.')
+
+    """ Indexing Step """
+
+    Index5 = range(8)
+    Index5rows = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H']
```

Comment:
optional, but str type is iterable (`index5rows = 'ABCDEFGH'`)
also, convention is lowercase first letter